src/pixelator/graph/backends/implementations.py

```diff
@@ -787,3 +627,196 @@ def add_names_to_vertexes(self, vs_names: List[str]) -> None:
                             of different length
         """
         raise NotImplementedError()
+
+
+class IgraphBasedVertexSequence(VertexSequence):
+    """Proxy for a igraph.VertexSeq."""
+
+    def __init__(self, raw) -> None:
+        """Instantiate a new IgraphBasedVertexSequence."""
+        self._raw = raw
+
+    def select(self, **kwargs):
```

Comment:
Yes, I've intentionally not set one here, because I'm planing to replace the igraph general select api with a more specific one. I'll keep this type-less for now.